web-src/src/components/App.js

```diff
@@ -16,13 +16,26 @@ import {
 } from '@adobe/react-spectrum';
 import { useRecoilValue, useRecoilState } from 'recoil';
 import { css } from '@emotion/css';
+import { IntlProvider, useIntl } from 'react-intl';
+
+import { intlMessages } from './App.l10n.js';
 import { ConsentDialog } from './ConsentDialog.js';
 import { MainSidePanel } from './MainSidePanel.js';
 import { PromptSessionPanel } from './PromptSessionPanel.js';
 import { PromptTemplateLibraryPanel } from './PromptTemplateLibraryPanel.js';
 import { viewTypeState, ViewType } from '../state/ViewType.js';
 import { mainSidePanelState, MainSidePanelType } from '../state/MainSidePanelState.js';
 import { FavoriteVariantListPanel } from './FavoriteVariantListPanel.js';
+import { useShellContext } from './ShellProvider.js';
+
+/* eslint-disable import/extensions */
+import * as messages1 from './__localization__/App.l10n';
```

Comment:
Can this be renamed to `messagesApp` and `messagesMainSidePanel` etc?

Reply:
We might simplify message loading with a webpack plugin, though it's not urgent. We can refactor this later.